### DISCUSSION TITLE
Contribute

### DISCUSSION THREAD

**Discussion by priyanshu-panwar** [br3ndonland/fastenv#17] [Ideas]
Hey I would love to contribute to this project. If there's anything new we can add, I am up for it.

**Comment by br3ndonland** [comment]
> Hey I would love to contribute to this project. If there's anything new we can add, I am up for it.

That would be great! I've had a good time developing this package so far, and I think it could be useful to people. A few things I'm planning for the near future include:

- br3ndonland/fastenv#18
- br3ndonland/fastenv#19
- br3ndonland/fastenv#20

Feel free to try working on any of that stuff. Even just providing feedback here in the discussions is helpful.

I'm also working on adding support for Cloudflare R2 for downloading/uploading dotenv files. It's pretty involved, and Cloudflare's API and docs are still pretty rough around the edges, so I'll probably tackle that myself.
